lib/p5.play.js

```diff
@@ -1263,6 +1293,19 @@ function Sprite(pInst, _x, _y, _w, _h) {
 
     if(!this.removed)
     {
+      if (this.collider) {
+        var vMagnitude = this.velocity.mag();
+        var vPerpendicular = createVector(this.velocity.y, -this.velocity.x);
+
+        this._sweptCollider.width = vMagnitude + 2 * this.collider._getRadiusOnAxis(this.velocity);
+        this._sweptCollider.height = 2 * this.collider._getRadiusOnAxis(vPerpendicular);
+        this._sweptCollider.rotation = radians(this.getDirection());
+        this._sweptCollider.center = createVector(
+          this.newPosition.x + 0.5 * this.velocity.x,
+          this.newPosition.y + 0.5 * this.velocity.y
+        );
```

Comment:
Not sure if this is just echoing Brad's comment, but it seems like this computation is probably not necessary on every frame, but only the ones where we need to access the _sweptCollider. I assume we're filtering out a lot of those potential checks with the quadtree elsewhere?